oneview/resource_fc_network.go

```diff
@@ -102,6 +115,12 @@ func resourceFCNetworkCreate(d *schema.ResourceData, meta interface{}) error {
 		Description: d.Get("description").(string),
 	}
 
+	rawInitialScopeUris := d.Get("initial_scope_uris").(*schema.Set).List()
```

Comment:
We should use `d.GetOk`  since `initial_scope_uris` is Optional and might not be set in the config file

Reply:
Implemented using GetOk